apps/image_augmentation/image_augmentation.cpp

```diff
@@ -262,21 +262,27 @@ int main(int argc, const char ** argv)
             color_temp_increment *= -1;
 
         rocalUpdateIntParameter(rocalGetIntValue(color_temp_adj)+color_temp_increment, color_temp_adj);
-
-        rocalCopyToOutput(handle, mat_input.data, h*w*p);
+        auto ouput_tensor_list = rocalGetOutputTensors(handle);
+        unsigned char* output = mat_input.data;
+        for (uint i = 0; i < ouput_tensor_list->size(); i++)
+        {
+            ouput_tensor_list->at(i)->copy_data(output);
+            output += ouput_tensor_list->at(i)->data_size();
+        }
         counter += inputBatchSize;
         if(!display)
             continue;
 
-        mat_input.copyTo(mat_output(cv::Rect(  col_counter*w, AMD_ROCm_Black_resize.rows, w, h)));
+        std::string out_filename = std::string(outName) + ".png"; 
+        mat_input.copyTo(mat_output(cv::Rect(col_counter * w, AMD_ROCm_Black_resize.rows, w, h)));
         if(color_format ==  RocalImageColor::ROCAL_COLOR_RGB24 )
         {
             cv::cvtColor(mat_output, mat_color, CV_RGB2BGR);
-            cv::imshow("output",mat_color);
+            cv::imwrite(out_filename, mat_color);
```

Comment:
we would like the image_augmentaton app to display the output images as before.

Reply:
We are displaying the image as before. Just changed the name to image_augmentation_app.png